docs/blog/unettest.md

```diff
@@ -0,0 +1,291 @@
+# Network Rules as (Tested) Code
+
+![What is the answer, O Deep Thought](https://docs-unettest.s3.us-east-2.amazonaws.com/o_deep_thought.png)
+
+When we write software, there are many ways to ask a question. We could go to
+Stack Overflow or to a smarter coworker, but I like to ask my computer first.
+There are many ways to ask computers questions, and one of the most effective is
+with TDD. TDD is awesome. We can ask many questions quickly, and once a question
+is asked, we can be confident it will remain answered. Given a question on how
+something should work, write a test to describe the behavior. Then run the tests
+and write code to make the test green. Great stuff. If there is a corner case, a
+test case can expose it. When code turns red, make it better.
```

Comment:
This paragraph feels a little to me like the intro paragraph of an essay. Analogous to "At the dawn of man, good an evil was born...". Too generic and devoid of substance. 
My advice would be to scrap the whole thing and add in the meat of this paragraph below your next paragraph. I'll specify in another comment.